content/blog/2025/02/camel410-whatsnew/index.md

```diff
@@ -0,0 +1,188 @@
+---
+title: "Apache Camel 4.10 What's New"
+date: 2025-02-12
+authors: [davsclaus, orpiske, tadayosi]
+categories: ["Releases"]
+preview: Details of what we have done in the Camel 4.10 release.
+---
+
+Apache Camel 4.10 LTS has just been [released](/blog/2025/02/RELEASE-4.10.0/).
```

Comment:
Why this link? It seems to me it will just self link this page.

Reply:
its gregors post